Lib/profile/sample.py

```diff
@@ -53,6 +57,14 @@ def sample(self, collector, duration_sec=10):
                     break
                 except (RuntimeError, UnicodeDecodeError, MemoryError, OSError):
                     errors += 1
+                except ValueError as e:
```

Comment:
I wonder if we should add an extra condition here and only do this if we are the ones who spawned the subprocess. Maybe this is fine as-is. What do you think @pablogsal?

Reply:
Uf, I am not a fun of this as this may be hiding errors in `collector.collect(stack_frames)`. We need some better signal here

Reply:
Actually, as long as `get_stack_trace` can raise a `ProcessNotFoundError` on all platforms, we shouldn't need this as that should capture this situation. We have to fix how this is handled on macOS then as we get a `ValueError` there.